tsdb/operations/default.json

```diff
@@ -86,4 +90,43 @@
           }
         }
       }
-    }
+    },
+    {
+      "name": "date-histo-with-time-series-1h",
+      "operation-type": "search",
+      {%- if ingest_mode is defined and ingest_mode == "data_stream" %}
+      "index": "k8s",
+      {%- else %}
+      "index": "tsdb",
+      {%- endif %}
+      "body": {
+        "size": 0,
+        "aggs": {
+          "by_timestamp": {
+            "date_histogram": {
+              "field": "@timestamp",
+              "fixed_interval": "1h"
+            },
+            "aggs": {
+              "ts": {
+                "time_series": {
+                  "keyed": false
+                },
+                "aggs": {
+                  "available_memory": {
+                    "min": {
+                      "field": "kubernetes.node.memory.available.bytes"
+                    }
+                  }
+                }
+              },
+              "min_available_memory": {
```

Comment:
I am not sure why we are also  including this pipeline aggregation. Do we expect to see changes in performance if we introduce some optimization for time_series aggregator?

Reply:
I added this because I think a pipeline aggregator is part of the usage pattern with time series aggregation.
All the time series buckets is an information overflow, getting min, max or derivative from all time series buckets will be useful informations to plot in a graph.

Reply:
Or do you think we should just focus on the time series agg itself in the benchmark? So removing the pipeline agg and maybe even the date histogram here? So it is just the time_series agg?